extensions/one-time-password/src/one-time-password.tsx

```diff
@@ -16,10 +16,19 @@ import {
 } from '@raycast/api';
 import { useEffect, useState } from 'react';
 import { getProgressIcon } from '@raycast/utils';
+import { z } from 'zod';
```

Comment:
I think we don't have a strong need for Zod for the current plugin, and adding Zod will only make the package size even bloated.